test_email_validation.py

```diff
@@ -0,0 +1,8 @@
+import email_validation
```

Comment:
usually stdlib modules are imported first and only after them you import your own project modules